src/Compilers/VisualBasic/Portable/Symbols/SignatureOnlyMethodSymbol.vb

```diff
@@ -162,7 +162,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Public Overrides ReadOnly Property TypeArguments() As ImmutableArray(Of TypeSymbol)
             Get
-                Throw ExceptionUtilities.Unreachable
+                Return ImmutableArray(Of TypeSymbol).Empty
```

Comment:
If we are going to implement this method, The implementation should be correct.

---
In reply to: [81688890](https://github.com/dotnet/roslyn/pull/14229#discussion_r81688890) [](ancestors = 81688890)